packages/core/upload/server/controllers/admin-folder-file.js

```diff
@@ -40,10 +40,13 @@ module.exports = {
 
     if (deletedFiles.length + deletedFolders.length > 1) {
       strapi.telemetry.send('didBulkDeleteMediaLibraryElements', {
-        rootFolderNumber: deletedFolders.length,
-        rootAssetNumber: deletedFiles.length,
-        totalFolderNumber,
-        totalAssetNumber: totalFileNumber + deletedFiles.length,
+        adminUser: ctx.state?.user,
+        groupProperties: {
```

Comment:
I'm not confident those data should be in the `groupProperties` 🤔
@petersg83 I can see `deletedFiles` or `deletedFolders` which means that those data are associated with this specific event only right?

Reply:
Even if it changes on an event, it will represent the state of how many folders/media is/are in the project at a given time, no? In my mind that falls under permanent group data

Reply:
No that's the thing the naming of those properties makes me say that they represent the number of deleted files and folders during the associated call.

Reply:
Just had a chat with him, and it's indeed what I thought it was. `rootFolderNumber` for instance is the number of folders selected when doing the bulk delete (thus deleted but only on the first level), and `totalFolderNumber` is the `rootFolderNumber` plus the number of folders that were nested inside the one the action deleted.

So as you can see this data is tightly linked to the event, it should be sent in the `eventProperties` putting it elsewhere will create confusion.